controllers/helmchart_controller.go

```diff
@@ -430,8 +435,6 @@ func (r *HelmChartReconciler) reconcileFromHelmRepository(ctx context.Context,
 
 		readyMessage = fmt.Sprintf("Fetched and packaged revision: %s", newArtifact.Revision)
 		readyReason = sourcev1.ChartPackageSucceededReason
```

Comment:
I think just adding a `break` statement here would solve the problem. The idea of the default block fallthrough was to prevent duplication.

Reply:
Yeah thats fine as well, I've changed it to just having a break here.